docs/how-to-create-a-stream.md

```diff
@@ -1,6 +1,6 @@
 # How to Create a Stream
 
-The Shared Signals and Events spec leaves it up to the implementer how to
+The Shared Signals spec leaves it up to the implementer how to
```

Comment:
should this be Shared Signals Framework spec?

Reply:
Meh. I think calling it "Shared Signals" sometimes is just fine